adapter/stdioLogger/stdioLogger.go

```diff
@@ -17,19 +17,36 @@
 package stdioLogger
 
 import (
-	"encoding/json"
-	"io"
-	"os"
+	"fmt"
 
-	me "github.com/hashicorp/go-multierror"
+	multierror "github.com/hashicorp/go-multierror"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 
 	"istio.io/mixer/adapter/stdioLogger/config"
 	"istio.io/mixer/pkg/adapter"
 )
 
 type (
 	builder struct{ adapter.DefaultBuilder }
-	logger  struct{ logStream io.Writer }
+	logger  struct {
+		outputPaths []string
+		impl        *zap.Logger
+	}
+	zapBuilderFn func(outputPaths ...string) (*zap.Logger, error)
+)
+
+const (
+	stdErr = "stderr"
+	stdOut = "stdout"
```

Comment:
They may not, really. I just tend to screw up typing a bunch and it is an easy thing to miss when (self-) reviewing, so i was trying to put them in one spot to eliminate my own tendency towards typos.

When this whole thing is rewritten in two weeks, we can revisit ;).